Alinawayne

alina-wayne opened the pull request:
Java homework 3.9.2024 - AW

alina-wayne committed:
Changes to main block

alina-wayne committed:
Changes to main block 1

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C to F ( C = 0)

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
implementing HoursToMinutesAndSecondsConvertor

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit

vikmaksymenko reviewed (changes requested):
Давайте коментарям до комітів осмислені назви, інакше складно буде зрозуміти, що і в якому ви робили, перш за все вам самій за тиждень-два 
![image](https://github.com/skillup-aqa/theme3-java/assets/5682824/bb61a188-26c7-469f-88e4-d0fb3d293899)

vikmaksymenko commented on src/main/java/part1/ArithmeticMeanCalculator.java:
```diff
@@ -1,16 +1,18 @@
-        // TODO: Implement solution here
-        return -1;
+    public static double calculateArithmeticMean(int a, double b, double c) {
+        // Calculate the mean
```
Коментарі до коду - це чудово, але тут назва дає достатньо контексту. Плюс якщо ви описуєте метод чи клас, краще користуватися [ Java Doc ](https://www.baeldung.com/javadoc)

vikmaksymenko commented on src/main/java/part1/ArithmeticMeanCalculator.java:
```diff
@@ -1,16 +1,18 @@
 
     public static void main(String[] args) {
-        double a = 1;
+        int a = 1;
```
А чому `a` став `int`?

vikmaksymenko commented on src/main/java/part1/CelsiusToFahrenheitConvertor.java:
```diff
@@ -1,14 +1,14 @@
-        return -1;
+
+    public static double celsiusToFahrenheitConvertor(double celsius) {
+     double fahrenheit = (celsius * 9/5) +32;
```
Підправте форматування 
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#formatter-in-settings

vikmaksymenko commented on src/main/java/part1/CelsiusToFahrenheitConvertor.java:
```diff
@@ -1,14 +1,14 @@
-        // TODO: Implement solution here
-        return -1;
+
+    public static double celsiusToFahrenheitConvertor(double celsius) {
```
А чому ви змінили назву методу?

vikmaksymenko commented on src/main/java/part1/HoursToMinutesConvertor.java:
```diff
@@ -1,21 +0,0 @@
-package part1;
-
-public class HoursToMinutesConvertor {
-    public static int convertHoursToMinutes(int hours) {
-        // TODO: Implement solution here
-        return -1;
-    }
-
-    public static int convertHoursToSeconds(int hours) {
-        // TODO: Implement solution here
-        return -1;
-    }
-
-    public static void main(String[] args) {
-        int hours = 1;
-        int result = convertHoursToMinutes(hours);
-        System.out.println(result);
-        result = convertHoursToSeconds(hours);
-        System.out.println(result);
-    }
-}
```
Думаю, ви дарма видалили цей файл, поверніть його і перенесіть рішення з 'HoursToMinutesAndSecondsConverter '

vikmaksymenko commented on src/main/java/part1/PercentCalculator.java:
```diff
@@ -1,15 +1,10 @@
-        System.out.println(result);
+        double total = 100;
+        double value = 10;
+        double percentage = (value / total) * 100;
```
Перенесіть будь-ласка рішення в метод 'calculatePercent'

vikmaksymenko commented on src/test/java/part1/HoursToMinutesConvertorAndSecondsTest.java:
```diff
@@ -1,10 +1,5 @@
 package part1;
 
-import org.testng.annotations.Test;
-import part1.HoursToMinutesConvertor;
-
-import static org.testng.Assert.assertEquals;
-
```
Цей код ви також дарма видалили

vikmaksymenko commented on src/main/java/part1/SalaryCalculator.java:
```diff
@@ -1,18 +1,16 @@
     public static void main(String[] args) {
-        System.out.println(calculateSalaryBeforeTaxes(160, 10));
-        System.out.println(calculateSalaryAfterTaxes(160, 10));
+    double hoursPerMonth = 160;
+    double hourlyRate = 10;
+    double ESV = 1562;
+    double taxRate = 0.95;
+    double calculateSalaryBeforeTaxes = hoursPerMonth * hourlyRate;
+    double calculateSalaryAfterTaxes = calculateSalaryBeforeTaxes * taxRate;
+    double resultSalaty = Math.round(calculateSalaryAfterTaxes - ESV);
```
Перенесіть цей код в відповідні методи

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java
Updating SalaryCalculator.java

alina-wayne commented:
Implementing changes for homework java-3

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java
Updating SalaryCalculator.java
Implementing a program that checking numbers on "odd" or "even"

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java
Updating SalaryCalculator.java
Implementing a program that checking numbers on "odd" or "even"
Implementing method for MaxOfThreeChecker.java

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java
Updating SalaryCalculator.java
Implementing a program that checking numbers on "odd" or "even"
Implementing method for MaxOfThreeChecker.java
Implementing solving method for LeapYearChecker

alina-wayne committed:
Changes to main block 1
Implementing solution for calculateArithmeticMean
Implementing C1 to F1 ( C = 0)
Implementing C2 to F2 ( C = 100)
Implementing C3 to F3 ( C = 0)
Implementing division program
Implementing HoursToMinutesAndSecondsConvertor
Implementing PercentCalculator
Implementing SalaryCalculator
test commit
Changing in ArithmeticMeanCalculator.java "double a"
Rename HoursToMinutesAndSecondsConvertor.java class
Update CelsiusToFahrenheitConvertor.java
Updated DivisionWithRemainderCalculator.java
Updating HoursToMinutesAndSecondsConverter.java
Updating PercentCalculator.java
Updating SalaryCalculator.java
Implementing a program that checking numbers on "odd" or "even"
Implementing method for MaxOfThreeChecker.java
Implementing solving method for LeapYearChecker
Implementing code for VowelChecker.java
